pipeline/metadata/install.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<COMPONENT_INSTALLATION_DESCRIPTOR>
+	<OS>
```

Comment:
Did you leave HARD TABS in OUR REPOSITORY

Reply:
Auto generated. Not my fault.